### ISSUE TITLE
2020-12-27

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#406]
481II.3 Experimental Setting
4821.optimized feedforward neural networks: one to five hidden layers, with one thousand hidden units per layer, and with a softmax logistic regression for the output layer; The cost function is the negative log-likelihood; learning rate is a hyperparameter; (2020-12-15)
4832.varied the type of non-linear activation function: sigmoid, hyperbolic, softsign; (2020-12-15)
4843.search for the best hyperparameters (learning rate and depth) separately for each model; (2020-12-15)
4854.Weights at each layer, uniform distribution in the interval (−a, a); (2020-12-15)
486
487III.Effect of Activation Functions and Saturation During Training
488III.1 Experiments with the Sigmoid
4891.Figure 2 shows the evolution of the activation values (after the nonlinearity) at each hidden layer; The graph shows the means and standard deviations of these activations; (2020-12-15)
